catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -103,8 +104,8 @@ public UpdateRequest process(UpdateRequest input)
 
     private <T> T validate(T item, Function<T, Metacard> itemToMetacard,
             Map<String, Integer> counter) {
-        Set<String> errors = new HashSet<>();
-        Set<String> warnings = new HashSet<>();
+        List<Serializable> errors = new ArrayList<>();
```

Comment:
For what it's worth, I don't think it's necessary to change the sets to lists in order to do what the PR summary claims to do. The old errors/warnings could be added to the set before they're added onto the metacard.